src/source/Ice/IceAgent.c

```diff
@@ -1902,7 +1902,7 @@ STATUS iceCandidateSerialize(PIceCandidate pIceCandidate, PCHAR pOutputData, PUI
                                  pIceCandidate->ipAddress.address[1],
                                  pIceCandidate->ipAddress.address[2],
                                  pIceCandidate->ipAddress.address[3],
-                                 (UINT16) getInt16(pIceCandidate->ipAddress.port),
+                                 (UINT16) SWAP_INT16(pIceCandidate->ipAddress.port),
```

Comment:
is this actually an issue for you? getInt16 will trigger SWAP_INT16 on LE machine.

Reply:
Oh.. I see. I assumed SWAP_INT16 is the one equivalent to ntohs().
Weird.. yes it was definitely swapped because the port number reported by this code path was deferent from what onIceCandidate() reported. By doing this, it worked. (SDP matched the port number it actually listening)

```
$ uname -a
Darwin xxxxx.local 19.4.0 Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64 x86_64
```

Reply:
What do we keep in memory? Is this adapted port or the on-the-wire port? I assume the spec is BE right?

Reply:
Again `initKvsWebRtc()` wasn't called in my code. I have already reverted the change. Let me resolve this conversation.